### PR TITLE
tasks.php: Use modern DPDatabase::query, escaping, sprintf

### PR DESCRIPTION
**Pull request by bpfoley**
Tested task lifecycle (creating, modifying, adding/removing comments, related task, deleting).
Sandbox at https://www.pgdp.org/~bfoley/c.branch/cleanup14

### CODE REVIEW

File: tasks.php [hunk 1]
```diff
@@ -743,8 +743,8 @@ function handle_action_on_a_specified_task()
                     percent_complete = $edit_percent
```
Comment (chrismiceli) [inline tasks.php:743]: do we want to remove the $ variables and use sprintf exclusively to avoid %s being in a string throwing things off?

Comment (bpfoley) [reply tasks.php]: Done.

**Comment by cpeel** [comment]
I beat you to this by 5 hours: https://github.com/DistributedProofreaders/dproofreaders/pull/485

Even put you on the reviewers list!

**Comment by bpfoley** [comment]
> I beat you to this by 5 hours: #485
> 
> Even put you on the reviewers list!

Derp! Oh well, at least it got done.

**Comment by cpeel** [comment]
Obviated by https://github.com/DistributedProofreaders/dproofreaders/pull/485

**Pull request closed without merging**